libvast/vast/format/pcap.hpp

```diff
@@ -101,6 +103,7 @@ class reader : public single_layout_reader {
   size_t snaplen_;
   bool community_id_;
   type packet_type_;
+  mutable pcap_stat last_stats_;
```

Comment:
Is this correctly default-initialized or does it need a `memset`?

Reply:
This is default-initialized in the constructor of the reader.